releases/point-2022-2-3-release-note.md

```diff
@@ -30,7 +30,7 @@ See detailed step-by-step instructions on how to upgrade SysKit Point in the [up
     * **If enabled, the Collaborator will be able to view** the OneDrive tile, the Analytics & Usage tile, and Audit Log Reports when accessing User details or workspaces they manage. 
   * For more details on Role-Based Access take a look at [this article](../configuration/enable-role-based-access.md).
 
-* **Fixed an issue** where archiving a Microsoft Team would result in the following error: 'Failed to execute Templates backend request ArchiveRequest'.
+* **Fixed an issue** where archiving a Microsoft Team would result in the following error: `Failed to execute Templates backend request ArchiveRequest`.
```

Comment:
vidim na starijim release notesima da je točka na kraju rečenice unutar zagrada .. pa da samo zamijeniš

`Failed to execute Templates backend request ArchiveRequest.`